kentik_api_library/kentik_api/api_connection/retryable_session.py

```diff
@@ -0,0 +1,25 @@
+import logging
+from requests import Session
+from requests.adapters import HTTPAdapter
+from requests.packages.urllib3.util.retry import Retry
+from typing import Optional
+
+log = logging.getLogger(__name__)
+
+# mypy: ignore-errors
+class RetryableSession(Session):
+    DEFAULT_RETRY_STRATEGY = Retry(
+        total=3,
+        backoff_factor=1,
+        status_forcelist=[429, 500, 502, 503, 504],
+        allowed_methods=["HEAD", "GET", "PUT", "OPTIONS", "POST"],
```

Comment:
Any particular reason why DELETE isn't retried by default?

Reply:
Good point. We should retry all operations including DELETE and PATCH. I'll fix it.